packages/odyssey-react-mui/src/SearchField.tsx

```diff
@@ -23,8 +23,14 @@ import {
 
 import { CloseCircleFilledIcon, SearchIcon } from "./icons.generated";
 import { Field } from "./Field";
+import { FieldComponentProps } from "./FieldComponentProps";
 import type { SeleniumProps } from "./SeleniumProps";
 
+type FieldComponentPropsUsedBySearchField = Pick<
```

Comment:
```suggestion
type SearchFieldFieldComponentProps = Pick<
```

I'm seeing too many places where we're doing an `Omit`. Instead, we should just `Pick` on each one.

And I don't think we need to make a new variable. I think inlining this type is fine.

```ts
} & Pick<FieldComponentProps, "id" | ...> & SeleniumProps
```

Reply:
I tried swapping back and forth between `Pick` and `Omit` depending on how many of the props we needed. If we needed all but 1 or 2, went with `Omit` if we only needed 1 or 2, went with `Pick`. Happy to go one way or the other. Was just trying to keep it as concise as possible